pagopa/pn-frontend
# Pn 6004 fe deleghe p giuridiche manca il nome del delegato e funziona copia codice

## Short description
Fixed issue with delegate name in CodeModal component in PG and added copy to clipboard button for verification code.

## List of changes proposed in this pull request
- Fixed issue with delegate name in CodeModal component in PG
- Added CopyToClipboardButton component in CodeModal to allow copy to clipboard of verification code. This affects to PF too
- Added new test for CodeModal

## How to test
Login PG and open delegates. Create a new delegation and look for "Show code" ("mostra codice") context menu. Now the modal should properly display the name of delegate. Also the clipboard button is added in this modal.
